packages/@aws-cdk/aws-scheduler-alpha/test/target.test.ts

```diff
@@ -0,0 +1,53 @@
+import { App, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { ScheduleExpression } from '../lib';
+import { Schedule } from '../lib/private';
+import { targets } from '../lib/target';
+
+describe('schedule target', () => {
+  let stack: Stack;
+  let func: lambda.IFunction;
+  const expr = ScheduleExpression.at(new Date(Date.UTC(1969, 10, 20, 0, 0, 0)));
+
+  beforeEach(() => {
+    const app = new App();
+    stack = new Stack(app, 'Stack', { env: { region: 'us-east-1', account: '123456789012' } });
+    func = lambda.Function.fromFunctionArn(stack, 'Function', 'arn:aws:lambda:us-east-1:123456789012:function/somefunc');
+  });
```

Comment:
Created role and stack in the same account. Otherwise `lambda.Function.grantInvoke` fails as it assumes that function is imported (from a different account).

Also, `grantInvoke` method creates permission on Lambda function itself (updates resource policy) if the role is in the different account.